### ISSUE TITLE
at-function-named-arguments for function in map

### ISSUE THREAD

**Issue by ysds** [kristerkari/stylelint-scss#289]
```
"scss/at-function-named-arguments": "never"
```

Like a following patterns are expected no errors, but errors.

```scss
$color: #123456;

$rgb: (
  "r": red($color),
  "g": green($color),
  "b": blue($color)
);
```

**Comment by kristerkari** [comment]
Pull request with a fix:
https://github.com/kristerkari/stylelint-scss/pull/290

**Comment by kristerkari** [comment]
Released the fix with 3.4.1:
https://github.com/kristerkari/stylelint-scss/releases/tag/3.4.1
